docs/apache-airflow-providers-mongo/connections/mongo.rst

```diff
@@ -0,0 +1,82 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+.. _howto/connection:mongo:
+
+MongoDB Connection
+==================
+
+The MongoDB connection type enables the MongoDB Integrations.
```

Comment:
I think you need to add MongoDB to spelling words too

Reply:
I believe adding mongodb lowercase satisfies upper case as well

Reply:
yes, if you doc build step is OK no problem. I said that because in spelling words list we have Capitalize and lowercase words =)